src/ui/backend/BaseBacking.js

```diff
@@ -62,36 +83,39 @@ export default class BaseBacking {
 
   copy () {
     var copy = {};
+    for (var key in BASE_STYLE_PROPS) {
+      copy[key] = this[key];
+    }
 
-    for (var i = 0; i < styleKeyList.length; i++) {
-      var key = styleKeyList[i];
+    for (var i = 0; i < extendedStylePropList.length; i++) {
+      var key = extendedStylePropList[i];
       copy[key] = this[key];
     }
 
     return copy;
   }
 
   update (style) {
-    for (var i = 0; i < styleKeyList.length; i++) {
-      var key = styleKeyList[i];
+    for (var key in style) {
+      if (this[key] !== undefined) {
```

Comment:
Oh, I confirmed that `void 0` only replaces `undefined` in release mode builds; you won't see that happen in the simulator, and it won't be necessary to use it here or in general.